cmd/secretd/cli_attestation.go

```diff
@@ -0,0 +1,110 @@
+// +build !secretcli
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const flagReset = "reset"
+
+func InitAttestation() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "init-enclave [output-file]",
+		Short: "Perform remote attestation of the enclave",
+		Long: `Create attestation report, signed by Intel which is used in the registation process of
+the node to the chain. This process, if successful, will output a certificate which is used to authenticate with the 
+blockchain. Writes the certificate in DER format to ~/attestation_cert
+`,
+		Args: cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			println("This is a secretd only function, yo")
```

Comment:
we should probably remove the "yo", yo 😛

Reply:
No, it's obviously necessary